src/displayapp/screens/Dice.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "displayapp/screens/Screen.h"
+#include "displayapp/widgets/Counter.h"
+#include "components/datetime/DateTimeController.h"
+#include <lvgl/lvgl.h>
+
+namespace Pinetime::Applications::Screens {
+  class Dice : public Screen {
+    public:
+      Dice(DisplayApp* app, Controllers::DateTime& dateTime);
+      ~Dice() override;
+      void Roll();
+      void NextColor();
+
+    private:
+      Controllers::DateTime& dateTime;
+
+      lv_obj_t* btnRoll;
+      lv_obj_t* btnRollLabel;
+      lv_obj_t* resultLabel;
+
+      lv_color_t resultColors[3] = {LV_COLOR_YELLOW, LV_COLOR_MAGENTA, LV_COLOR_AQUA};
```

Comment:
you could use a `std::array<lv_color_t, 3>`. Then you won't need the `resultColorsLength` variable in the next line

Reply:
addressed in 3bd3d817a04134a50d11062de7c836e6a5251d2a